README.md

```diff
@@ -38,7 +38,59 @@ View operator status:
 Inspect the status of the `machineconfigpool` objects which track upgrades:
 `oc describe machineconfigpool`
 
-(More to come)
+# Applying configuration changes to the cluster
+
+The MCO has "high level" knobs for some components of the cluster state;
+for example, SSH keys and kubelet configuration.  However, there are
+obviously a quite large number of things one may want to configure on
+a system.  For example, offline environments may want to specify
+an internal NTP pool.  Another example is static network configuration.
+By providing a MachineConfig object containing Ignition, arbitrary
+files can be laid down into `/etc` and `/var`.
+
+One known ergonomic issue right now is that you must encode file contents
+via [`data:` URIs](https://en.wikipedia.org/wiki/Data_URI_scheme). This is part
+of the specification of the [Ignition project](https://github.com/coreos/ignition).
+
```

Comment:
Might be worth also mentioning we support laying down systemd units, rather than falling back to writing the units as files.